tools/publisher/src/subcommand/publish.rs

```diff
@@ -114,7 +123,7 @@ async fn publish(handle: &PackageHandle, crate_path: &Path) -> Result<()> {
 
 async fn confirm_correct_tag(location: &Path) -> Result<()> {
     let repository = Repository::new(SDK_REPO_NAME, location)?;
-    let versions_manifest = VersionsManifest::from_file(location.join("versions.toml"))?;
+    let versions_manifest = VersionsManifest::from_file(location.join("../versions.toml"))?;
```

Comment:
This was a bug. It was looking for `versions.toml` in `aws-sdk-rust/sdk` rather than in the repository root since the location is from the resolved publish root, which is the `sdk` directory.